src/lib/math/pcurves/pcurves_impl/pcurves_impl.h

```diff
@@ -458,6 +458,120 @@ class IntMod final {
       */
       constexpr Self invert() const { return pow_vartime(Self::P_MINUS_2); }
 
+      /**
+      * Helper for variable time BEEA
+      *
+      * Note this function assumes that its arguments are in the standard
+      * domain, not the Montgomery domain. invert_vartime converts its argument
+      * out of Montgomery, and then back to Montgomery when returning the result.
+      */
+      static constexpr void _invert_vartime_div2_helper(Self& a, Self& x) {
+         constexpr auto INV_2 = p_div_2_plus_1(Rep::P);
+
+         while((a.m_val[0] & 1) != 1) {
+
+            shift_right<1>(a.m_val);
```

Comment:
```suggestion
         while((a.m_val[0] & 1) != 1) {
            shift_right<1>(a.m_val);
```

... clang-format being pedantic.